network/bitcoin_transaction.ksy

```diff
@@ -0,0 +1,147 @@
+meta:
+  id: bitcoin_transaction
+  title: Bitcoin Transaction
+  license: CC-BY-SA-4.0
+  ks-version: 0.1.0
+  endian: le
+doc-ref: |
+  https://bitcoin.org/en/developer-guide#transactions
+  https://en.bitcoin.it/wiki/Transaction
+seq:
+  - id: version
+    type: u4
+    doc: |
+      Version number.
+  - id: nb_of_vins
+    type: u1
+    doc: |
+      Number of input transactions.
+  - id: vins
+    type: vin
+    repeat: expr
+    repeat-expr: nb_of_vins
+    doc: |
+      Input transactions.
+      An input refers to an output from a previous transaction.
+  - id: nb_of_vouts
+    type: u1
+    doc: |
+      Number of output transactions.
+  - id: vouts
+    type: vout
+    repeat: expr
+    repeat-expr: nb_of_vouts
+    doc: |
+      Output transactions.
+  - id: locktime
+    type: u4
+types:
+  vin:
+    seq:
+      - id: txid
+        size: 32
+        doc: |
+          Previous transaction hash.
+      - id: output_id
+        type: u4
+        doc: |
+          ID indexing an ouput of the transaction refered by txid.
+          This output will be used as an input in the present transaction.
+      - id: script_len
+        type: u1
+        doc: |
+          ScriptSig's length.
+      - id: script_sig
+        size: script_len
+        type: script_signature
+        doc: |
+          ScriptSig.
+        doc-ref: 'https://en.bitcoin.it/wiki/Transaction#Input'
+        doc-ref: 'https://en.bitcoin.it/wiki/Script'
+      - id: end_of_vin
+        contents: [0xff, 0xff, 0xff, 0xff]
+        doc: |
+          Magic number indicating the end of the current input.
+  vout:
+    seq:
+      - id: amount
+        type: u8
+        doc: |
+          Number of Satoshis to be transfered.
+      - id: script_len
+        type: u1
+        doc: |
+          ScriptPubKey's length.
+      - id: script_pub_key
+        size: script_len
+        doc: |
+          ScriptPubKey.
+        doc-ref: 'https://en.bitcoin.it/wiki/Transaction#Output'
+        doc-ref: 'https://en.bitcoin.it/wiki/Script'
```

Comment:
Unfortunately, YAML does not work like that: you can't really have multiple elements with the same name. This one will result in only one of these doc-refs to be used by ksc.

Reply:
In these cases I usually put a multiline string there, each line is a separate reference.

Reply:
Indeed, I didn't realise my mistake. I will fix it :+1:

Reply:
Fixed.